core/src/main/java/org/keycloak/jose/jws/crypto/HashUtils.java

```diff
@@ -30,15 +32,20 @@
  */
 public class HashUtils {
 
-    // See "at_hash" and "c_hash" in OIDC specification
-    public static String oidcHash(String jwtAlgorithmName, String input) {
+    // See:
+    // - "at_hash" and "c_hash" in OIDC specification (full = false)
+    // - "ath" in DPoP specification (full = true)
+    public static String oidcHash(String jwtAlgorithmName, String input, boolean full) {
```

Comment:
"ath" is not relevant to OIDC. Considering this point, the method name might not fit this usage.

Reply:
Agreed; how about `accessTokenHash()` instead?

Reply:
+1

Reply:
Done